### PR TITLE
Ensure correlation id is sent to SB

### PR DESCRIPTION
**Pull request by Korgath**
## References

* #0000

### CODE REVIEW

File: source/B2B.Transactions.Infrastructure/Configuration/ServiceBus/ServiceBusMessageMetadata.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace B2B.Transactions.Infrastructure.Configuration.ServiceBus
+{
+    public class ServiceBusMessageMetadata
+    {
+        public string? CorrelationID { get; set; }
+    }
```
Comment (kft) [inline source/B2B.Transactions.Infrastructure/Configuration/ServiceBus/ServiceBusMessageMetadata.cs:20]: Can it really be null ?

**Review by kft — CHANGES_REQUESTED** [review]
We need to move middleware to host-layer

**Review by MrDach — APPROVED** [review]

File: source/B2B.Transactions.Infrastructure/Transactions/TransactionQueueDispatcher.cs [hunk 2]
```diff
@@ -26,13 +27,16 @@ namespace B2B.Transactions.Infrastructure.Transactions
 {
     public class TransactionQueueDispatcher : ITransactionQueueDispatcher
     {
+        private const string CorrelationId = "Correlation-ID";
```
Comment (Mathiesen) [inline source/B2B.Transactions.Infrastructure/Transactions/TransactionQueueDispatcher.cs:30]: This will always be "Correlation-ID"???

Comment (Mathiesen) [reply source/B2B.Transactions.Infrastructure/Transactions/TransactionQueueDispatcher.cs:30]: nvm

**Review by Mathiesen — APPROVED** [review]

**Pull request merged**